test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
```

Comment:
Yeah, though, the week this is released, I bet someone wants the cascading functionality you have currently.

So... idk. I'm not a huge fan of all the configuration options, but as long as there are thorough tests for them, I think It could be ok.